### DISCUSSION TITLE
Erro ao tentar acessar os storages criados no azure, pelo Microsoft Azure Storage

### DISCUSSION THREAD

**Discussion by RafaelDifini** [balta-io/forum#99] [Tira Dúvidas] [answered]
##RESOLVIDO##

Criei meu storage conforme o curso, baixei o app, e conectei na minha conta, porem não consigo acessar meus recursos e não encontra minha assinatura, ja tentei logar diversas vezes, eu me autentico, porem nao muda.
![image](https://user-images.githubusercontent.com/79666617/201436495-48e3ec21-115e-4098-b410-b31bdd533db1.png)

![image](https://user-images.githubusercontent.com/79666617/201436527-972b6192-5f40-468b-b649-0c3bde062fa5.png)

alguem ja passou por isso? 

EDIT: Resolvido, era só marcar a opção "Diretorio Padrão"

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @RafaelDifini, como vai? Seleciona "Diretório Padrão" que deve aparecer.
